package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "wagtail",
+  "version": "1.0.0",
```

Comment:
Yeah, the version number here is meaningless, but keeping it in sync with the real version is a good idea regardless. The version can be updated programmatically by running `npm version 1.2.3` in the package directory, so if you have any sort of automated release helper, it could be done automatically.